packages/core/src/types.ts

```diff
@@ -285,6 +286,16 @@ export class Config {
      * @type {string|undefined}
      */
     this.userAgent = undefined
+
+    /**
+     * Specify if telemetry collection is disabled.
+     *
+     * Metrics are only collected when enabled both in server and driver instances.
+     *
+     * **Default**: ```false```
+     * @type {boolean}
```

Comment:
```suggestion
     * **Default**: ```false```
     * 
     * @type {boolean}
```
uniform formatting